testsuites/CBLTester/System_test_multiple_device/test_system_testing.py

```diff
@@ -0,0 +1,283 @@
+    password = "password"
+
+    # Create CBL database
+    sg_client = MobileRestClient()
+
+    doc_ids = set()
+    docs_per_db = num_of_docs / len(cbl_db_list)  # Equally distributing docs to db
+    extra_docs = num_of_docs % len(cbl_db_list)  # Docs left after equal distribution
+    num_of_itr_per_db = docs_per_db / num_of_docs_in_itr  # iteration required to add docs in each db
+    extra_docs_in_itr_per_db = docs_per_db % num_of_docs_in_itr  # iteration required to add docs leftover docs per db
+
+    c = cluster.Cluster(config=cluster_config)
+    if not resume_cluster:
+        # Reset cluster to ensure no data in system
+        c.reset(sg_config_path=sg_config)
+        sg_client.create_user(sg_admin_url, sg_db, username, password, channels=channels_sg)
+
+        # adding bulk docs to each db
+        for cbl_db, db_obj, db_name in zip(cbl_db_list, db_obj_list, db_name_list):
+            print "Adding doc on {} db".format(db_name)
+            doc_prefix = "cbl_{}".format(db_name)
+            j = 0
+            for j in range(num_of_itr_per_db):
+                ids = db_obj.create_bulk_docs(num_of_docs_in_itr, doc_prefix, db=cbl_db, channels=channels_sg, id_start_num=j * num_of_docs_in_itr, generator=generator)
+                x = ["{}_{}".format(doc_prefix, doc_id) for doc_id in range(j * num_of_docs_in_itr, (j * num_of_docs_in_itr) + num_of_docs_in_itr)]
+                assert sorted(ids) == sorted(x)
+                doc_ids.update(ids)
+            # adding remaining docs to each db
+            if extra_docs_in_itr_per_db != 0:
+                ids = db_obj.create_bulk_docs(extra_docs_in_itr_per_db, "cbl_{}".format(db_name), db=cbl_db, channels=channels_sg, id_start_num=(j + 1) * num_of_docs_in_itr, generator=generator)
+                x = ["{}_{}".format(doc_prefix, doc_id) for doc_id in range((j + 1) * num_of_docs_in_itr, ((j + 1) * num_of_docs_in_itr) + extra_docs_in_itr_per_db)]
+                assert sorted(ids) == sorted(x)
+                doc_ids.update(ids)
+        # add the extra docs to last db
+        if extra_docs != 0:
+            ids = db_obj.create_bulk_docs(extra_docs, "cbl_{}".format(db_name), db=cbl_db, channels=channels_sg, id_start_num=docs_per_db, generator=generator)
+            x = ["{}_{}".format(doc_prefix, doc_id) for doc_id in range(docs_per_db, docs_per_db + extra_docs_in_itr_per_db)]
+            assert sorted(ids) == sorted(x)
+            doc_ids.update(ids)
+    else:
+        # getting doc ids from the dbs
+        _check_doc_count(db_obj_list, cbl_db_list)
+        count = db_obj_list[0].getCount(cbl_db_list[0])
+        itr_count = count / query_limit
+        if itr_count == 0:
+            itr_count = 1
+        for _ in range(itr_count):
+            existing_docs = db_obj_list[0].getDocIds(cbl_db_list[0], query_limit, query_offset)
+            doc_ids.update(existing_docs)
+            query_offset += query_limit
+        print "{} Docs in DB".format(len(doc_ids))
```

Comment:
you can remove or you can use log_info if you need this to be logged

Reply:
changed it to log_info